### ISSUE TITLE
Headings - Indicates a heading even when set as display:none

### ISSUE THREAD

**Issue by ArnaudDelafosse** [Khan/tota11y#121]
Is it relevant to show a heading in the tree when no-one sees it (neither sighted users nor AT)?

I think it's misleading, unless it is indicated as invisible (display:none / visibility:hidden / aria-hidden:true) by using a different styling:
- green is "OK"
- red is "Error"
- black bkgd + white number could be "hidden to all"
- blue bkgd + white number could be "hidden to AT"?

**Comment by ArnaudDelafosse** [comment]
Is this project not maintained anymore?

**Comment by EcutDavid** [comment]
Agree, when elements are being marked as `display: none` or `aria-hidden="true"`, they are intended to be hidden from the readers.

**Comment by ArnaudDelafosse** [comment]
This is not maintained anymore. Too bad :/
